### ISSUE TITLE
[bug] Typing hint for adding multiple attributes incorrect

### ISSUE THREAD

**Issue by jwhonce** [mm21/trilium-alchemy#2]
```python
   task = Note(...)
    task += [Label("tag", t) for t in tags or []]
```
Works as expected but vscode displays 
```
Operator "+=" not supported for types "Note" and "list[Label]"
Operator "+" not supported for types "Note" and "list[Label]"
```

**Comment by jwhonce** [comment]
The Note.content field also reports a type error when assigned a string. Code works as expected.

**Comment by mm21** [comment]
Thanks for catching this. The `+=` operator should be fixed by #1 - I think it's because `__iadd__` wasn't returning a `Note` type.
